menu_after_form/media_queries_menuHTML/phone_screen.css

```diff
@@ -0,0 +1,82 @@
+@media (min-width: 320px) and (max-width: 540px) {
+	.data {
+		display: none;
+	}
+	.container-body {
+		display: flex;
+		justify-content: center;
+		align-items: center;
```

Comment:
phone_screen.css and tablet_screen.css are very similar, everything should be stored in one file